test/go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	k8s.io/utils v0.0.0-20231127182322-b307cd553661
 	sigs.k8s.io/cluster-api v0.0.0-00010101000000-000000000000
 	sigs.k8s.io/controller-runtime v0.18.2
-	sigs.k8s.io/kind v0.22.0
+	sigs.k8s.io/kind v0.23.0
```

Comment:
Let's please add kind to the ignore list in dependabot.yaml

I think it's better if we only do controlled bumps of this dependency